src/app/static/src/app/components/projects/ProjectHistory.vue

```diff
@@ -29,6 +31,7 @@
             <b-collapse :id="`versions-${p.id}`">
                 <div v-for="v in p.versions" :id="`v-${v.id}`" class="row font-italic bg-light py-2">
                     <div class="col-md-4 version-cell"></div>
+                    <div class="col-md-1 version-cell">{{`v${v.versionNumber}`}}</div>
```

Comment:
I'm getting 'vundefined' as the version number when I run the code myself here. I'll update my Docker and try again.

Reply:
Nevermind, updating my Docker fixed it.

Reply:
It might be a good idea to make the version numbers also reflect the project id, which is also an integer that increments by one each time. Something like `v.${p.id}.${v.versionNumber}`. You wouldn't necessarily have to change anything in the back end as the 'true' version number could stay the same but the user is shown this adapted version number with the project id prefix.

Reply:
I do like that idea because you do get unique 'v' ids then for each version, but I think it might be confusing for the user because they've never seen the project id anywhere else. Just having v1, v2, for each project seems simple and intuitive to me. @hillalex any thoughts?

Reply:
So projects also have a numeric id? For some reason I thought their names were primary keys. Agree adding more (seemingly to the user) random numbers in probably not worth it. I could image something like "project-name.v1" could be nice, but then again it's a lot of duplication. I reckon leave as `v1` etc for now and see what Jeff/users say. 

I have a feeling they might want to edit those version names themselves rather than just have them as auto-incremented numeric ids - v1, v2 etc is an improvement over no ids, but can imagine they still might be confusing (e.g. v1 ends up being the most recently updated, even though it was created first, could be a confusing user experience.)

Reply:
Yeah, I suspect Jeff will want the user to be able to name the versions themselves to reflect what they are doing. As Alex points out, semantic non-ordinal names would be best, but we should leave as is for now and collect feedback.